tests/suites/constraints/constraints.sh

```diff
@@ -0,0 +1,60 @@
+run_constraints_aws() {
+  # Echo out to ensure nice output to the test suite.
+  echo
+
+  file="${TEST_DIR}/constraints-aws.txt"
+
+  ensure "constraints-aws" "${file}"
+
+	echo "Deploy 5 machines with different constraints"
+  juju add-machine --constraints "root-disk=16G"
+  juju add-machine --constraints "cores=4"
+  juju add-machine --constraints "cpu-power=30"
+  juju add-machine --constraints "root-disk=16G cpu-power=30"
+  juju add-machine --constraints "instance-type=i3.xlarge"
```

Comment:
Is there a reason you picked such a chunky instance was chosen?

Reply:
1. I was inspired by our python-based tests
2. During local testing, I sometimes had issues with deploying 1 core machine on AWS. (4 cores were closest that differ from standard constraint which is equal to 2)